src/main/java/frc/robot/subsystems/ChassisSubsystem.java

```diff
@@ -94,55 +95,87 @@
 
   /** Creates a new ChassisSubsystem. */
   public ChassisSubsystem() {
-    rightMotors_.setInverted(true);
     frontLeftMotor_.setIdleMode(IdleMode.kCoast);
     frontRightMotor_.setIdleMode(IdleMode.kCoast);
     rearLeftMotor_.setIdleMode(IdleMode.kCoast);
     rearRightMotor_.setIdleMode(IdleMode.kCoast);
 
+    // set motor following
+    rearLeftMotor_.follow(frontLeftMotor_);
+    rearRightMotor_.follow(frontRightMotor_);
+
     frontLeftEncoder_ = frontLeftMotor_.getEncoder();
     frontRightEncoder_ = frontRightMotor_.getEncoder();
     rearLeftEncoder_ = rearLeftMotor_.getEncoder();
     rearRightEncoder_ = rearRightMotor_.getEncoder();
+
+    leftMotors_ = new MotorControllerGroup(frontLeftMotor_, rearLeftMotor_);
```

Comment:
I may be wrong, but if you are setting the rear motors to follow the front motors, then I don't think you should be putting them into a MotorControllerGroup. I believe that you should be using the Front controllers as they are as parameters to the DifferentialDrive call. Otherwise they are set to follow and will still get commands separately. 

My understanding is that if a motor is used in follow mode then you can't retrieve the Encoder info, so we might not want to use follow mode. On the other hand, since they are physically attached to one another, will the front and rear motors ever have different RPMs?

Reply:
This is all the documentation there is about SparkMax follow method -  I recall that you have recommended to set these. We have not had a chance to test this code with the encoder again because the Everybot was pulled apart. We could remove it since I don't know what is the behavior with this setting. I read the following as to mirror the output and it did say anything about that you can't use it in the MotorControllerGroup.

Causes this controller's output to mirror the provided leader.
Only voltage output is mirrored. Settings changed on the leader do not affect the follower.

Following anything other than a CAN SPARK MAX is not officially supported.